docs/development.md

```diff
@@ -16,6 +16,34 @@ By default, this script will deploy a `postgresql` instance, the worker and the
 
 You can skip deployment and basic settings if you use the script.
 
+### MacOS Users
+
+If you are using MacOS to develop for OpenScanHub you will need to set up a different environment:
```

Comment:
Nitpick: `s/MacOS/macOS` as that seem to be name capitalisation that Apple uses: https://www.apple.com/macos/ventura/

Reply:
I updated this in new commit

Reply:
Resolved in 82f4f94.